### PR TITLE
DDBCMS-90 - Full MasterKey Implementation

### PR DESCRIPTION
**Pull request by bozeak**

### CODE REVIEW

File: js/mkdru_ding.js [hunk 1]
```diff
@@ -0,0 +1,215 @@
+(function ($) {
+  mkdru.search = function () {
+    var filter = null;
+    var limit = null;
+    var limits = [];
+    var filters = [];
+    // prepare filter and limit parameters for pz2.search() call
```
Comment (NicolaeGrigoriu) [inline js/mkdru_ding.js]: Comments are not displayed based on Drupal Coding Standards.

File: js/mkdru_ding.js [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+
+  function get_advanced_search_params() {
+    var query = window.location.search;
+    if (query.length === 0) {
+      return false;
+    }
+    query = decodeURIComponent(query);
+    var vars = query.split("&");
+    var adv_q = [];
+    for (var i=0;i<vars.length;i++) {
```
Comment (NicolaeGrigoriu) [inline js/mkdru_ding.js]: Add necessary spaces.

File: js/mkdru_ding.js [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+        $(document).trigger('mkdru.onrecord', [data]);
+      },
+    };
+    if (mkdru.settings.mergekey) pz2Params.mergekey = mkdru.settings.mergekey;
+    if (mkdru.settings.rank) pz2Params.rank = mkdru.settings.rank;
+    if (mkdru.settings.sp_server_auth) pz2Params.pazpar2path += ';jsessionid=' + Drupal.settings.mkdru.jsessionid;
+    mkdru.pz2 = new pz2(pz2Params);
+    mkdru.pz2.showFastCount = 1;
+    // callback for access to DOM and pz2 object pre-search
+    for (var i=0; i < mkdru.callbacks.length; i++) {
```
Comment (NicolaeGrigoriu) [inline js/mkdru_ding.js]: Missing space.

**Comment by guddo** [comment]
You should `git mv` the JS files before doing any commits to these files.
It's impossible to review/compare changes.

**Pull request merged**